be/src/vec/exec/format/table/iceberg_reader.cpp

```diff
@@ -139,12 +159,32 @@ Status IcebergTableReader::init_row_filters() {
 
                 auto& delete_file_schema = metadata->schema();
                 vector<std::string> names;
-                for (auto i = 0; i < delete_file_schema.size(); ++i) {
+                int num_of_col = delete_file_schema.size();
```

Comment:
All delete files have same schema. Maybe we can just read one of them to get file schema.
To avoid read all files' metadata duplicately.